src/js/components/entry/License.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Markdown from 'react-markdown';
+import Icon from '../Icon';
+
+const OS_LICENSES = {
+  'Apache-1.1': 'Apache Software License Version 1.1',
+  'Apache-2.0': 'Apache Software License Version 2.0',
+  'BSD-2': 'Simplified BSD License (BSD 2-Clause License)',
+  'BSD-3': 'New BSD License (BSD 3-Clause License)',
+  'BSD-4': 'Original BSD License (BSD 4-Clause License)',
+  'BSD-style': 'BSD-style license, see project license for details',
+  CC0: 'Creative Commons Public Domain',
```

Comment:
Shouldn't these be in single quotations also?

Reply:
"CC0"

Reply:
"MIT" "Public"

Reply:
Not unless we disable the eslint rule for that line. If the value doesn't need quotes, the rule says leave them out.

Reply:
Ok, I see! Thank you!